docs/app/insights.md

```diff
@@ -8,19 +8,25 @@
 
 ## Browsing Dashboards
 
+![Dashboard Overview](../assets/app/dashboard-overview.png)
+
 Clicking into the Insights module immediately presents a listing of all dashboards you have access to. These are shown
 in the navigation bar and a listing on the main page.
 
 ## Creating a Dashboard
 
-1. Click the "Create Dashboard" action button (`+`) in the header.
+<video loop muted alt="Creating a Dashboard">
+  <source src="../assets/app/create-dashboard.mp4" type="video/mp4">
```

Comment:
Did we want to upload these somewhere centralized to reduce bundle sizes @benhaynes? 🤔

Reply:
Yeah, up to you if the images stay in the repo for fast loading and always being there (reliable fallback). But I think we should set up a long-time media S3 server (wrapped with CDN?) with versioned folders for these larger videos. They can load async in the Docs.

Maybe this same media server can also be used for Cloud image transformations? I'd like to use the directus.media TLD for all of this.

Reply:
Why not move the docs to separate repo? 
Advantage is size is not a limit and maintaining multiple version of assets in S3 and linking back to docs can be avoided?

Reply:
We follow a "monorepo" approach, and the docs are actually included (available to view) within the app, so this doesn't get around the issue since we'd need to load them in anyway.

Reply:
> Maybe this same media server can also be used for Cloud image transformations? I'd like to use the directus.media TLD for all of this.

There won't be an active web server handling those requests, it'd be just a static file storage solution (like S3). Also not sure what you mean by using it for cloud image transformations? Image transformations are handled by Directus itself, there's no standalone transformations server of any kind.

Reply:
Yeah! That works 🙂